backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5701720_sys_gh16164_C_Invoice_V.sql

```diff
@@ -0,0 +1,47 @@
+CREATE VIEW de_metas_endcustomer_fresh_reports.c_invoice_v
+            (
+             value,
```

Comment:
Why are you adjusting this?

Reply:
`c_invoice_v` depend on `de_metas_endcustomer_fresh_reports.hasFreeEUTaxes` so i need to drop this view before updating the function, and then create it egain

Reply:
@riadmen but there is no drop. You just recreate the view.
Pls, do that is the same file with function:
Drop the view, drop the function, recreate the function, recreate the view.